resources/js/features/root/pages/AdminFacilitiesList.page.tsx

```diff
@@ -18,31 +18,16 @@ export default (() => {
         staticPrefixQueryKey={Admin.keys.facility()}
         staticEntityURL="admin/facility"
         staticTranslations={createTableTranslations("facilities")}
-        intrinsicColumns={["id"]}
-        initialColumnsOrder={["id", "name", "url", "createdAt", "updatedAt"]}
-        initialVisibleColumns={["name", "url", "createdAt", "actions"]}
-        initialSort={[{id: "name", desc: false}]}
-        additionalColumns={["actions"]}
-        columnOptions={{
-          name: {
-            metaParams: {canControlVisibility: false},
-          },
-          url: {
-            columnDef: {
-              cell: (c) => `/${c.getValue()}`,
-            },
-          },
-          createdAt: {
-            columnDef: {
-              sortDescFirst: true,
-            },
-          },
-          updatedAt: {
-            columnDef: {
-              sortDescFirst: true,
-            },
-          },
-          actions: {
+        columns={[
+          {name: "id", initialVisible: false},
+          {name: "name", columnDef: {enableHiding: false}},
+          {name: "url", columnDef: {cell: cellFunc<string>((url) => `/${url}`)}},
+          {name: "createdAt", columnDef: {sortDescFirst: true}},
+          {name: "updatedAt", columnDef: {sortDescFirst: true}, initialVisible: false},
+          {
+            name: "actions",
+            isDataColumn: false,
```

Comment:
Czy to rozróżnienie isDataColumn potrzebne? Czemu nie samo dataColumns zamiast isDataColumn+extraDataColumns ? Coś w stylu:

domyślnie: dataColumns = [name]
np. actions: dataColumns: ["id"]
a jak będzie potrzeba to puste dataColumns:  [], albo kilka dataColumns: [a, b]

Reply:
Dlatego że kolumna ma swój typ i nullowalność wzięte ze schemy, i użyte na przykład do wyświetlenia filtra. Więc istnieje jedna główna kolumna z tquery którą dana kolumna tabeli wyświetla. Dodatkowo bardzo bym nie chciał walić się z kolumnami które nazywają się inaczej w tquery a inaczej w tabeli bo to niepotrzebne zamieszanie. W rezultacie wyszedł boolean.

Zacząłem od mniej więcej tego co piszesz, a potem stopniowo doszedłem do tego co jest :)